cube/helm/cube/templates/schema.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cube-schema
+data:
+{{ (.Files.Glob "schemas/**.yaml").AsConfig | indent 2 }}
```

Comment:
I think we might need to move these to the values file since users can't customize the file imports directly in the wrapper chart

Reply:
Yes, I've added documentation explaining how to add personal models. They will create another configmap which will be referenced in values.yaml
This one is just an example (same as the default user deployment for dagster)

Reply:
that's probably fair actually, i probably didn't tie in the docs reading the PR